gammapy/maps/hpxmap.py

```diff
@@ -111,29 +111,57 @@ def from_hdulist(cls, hdulist, hdu=None, hdu_bands=None):
             Map object
         """
         if hdu is None:
-            hdu = find_bintable_hdu(hdulist)
+            hdu_out = find_bintable_hdu(hdu_list)
         else:
-            hdu = hdulist[hdu]
+            hdu_out = hdu_list[hdu]
 
         if hdu_bands is None:
-            hdu_bands = find_bands_hdu(hdulist, hdu)
+            hdu_bands = find_bands_hdu(hdu_list, hdu_out)
 
+        hdu_bands_out = None
         if hdu_bands is not None:
-            hdu_bands = hdulist[hdu_bands]
+            hdu_bands_out = hdu_list[hdu_bands]
 
-        return cls.from_hdu(hdu, hdu_bands)
+        return cls.from_hdu(hdu_out, hdu_bands_out)
 
-    def to_hdulist(self, **kwargs):
-        hdu = kwargs.get('hdu', 'SKYMAP')
-        hdu_bands = kwargs.get('hdu_bands', 'BANDS')
-        hdu_out = self.make_hdu(**kwargs)
+    def to_hdulist(self, hdu='SKYMAP', hdu_bands=None, sparse=False, conv=None):
+        """Make a FITS `` with input data.
```

Comment:
Something is missing here in the backticks. Maybe just put this?
```
Convert to `~astropy.io.fits.HDUList`.
```

Reply:
Yes exactly.